sdk/resource/resource_test.go

```diff
@@ -205,6 +205,17 @@ func TestMerge(t *testing.T) {
 	}
 }
 
+func TestEmpty(t *testing.T) {
+	var res *resource.Resource
+	require.Equal(t, "", res.SchemaURL())
```

Comment:
I believe that you can use `assert` instead of `require` here and in the following lines

Reply:
Done.